package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "start": "cd server && node server.js",
     "dev": "npm --prefix server run dev",
-    "build": "npm --prefix server run build:ui",
+    "build": "npm --prefix client run build",
```

Comment:
@JoeSangine can you explain this change? I think this may cause issues with production.

Reply:
So the backend doesn't actually have the code to build, the backend was telling the frontend to build - but was being told by the root to build. So insted of root -> backend -> frontend to build, we say root -> frontend to build